scripts/hel-gis-data-variables.yaml

```diff
@@ -0,0 +1,21 @@
+environment:
+  dev:
+    fileshare: http://edit-here/fileshare-for-dev
+  test:
+    fileshare: http://edit-here/fileshare-for-test
+
+polys_dir: tormays_polys
+
+files: |
+  tormays_buses_polys.gpkg
```

Comment:
Olisi kätevämpää, jos tämä olisi oikea yaml-lista, eikä yksi iso string. Näitä rivejä voisi esim. kommentoida pois väliaikaisesti.

Reply:
Will make a new ticket for this.